File: Java/Car.java
class Car{
    Integer id;
    String licence;
    Account driver;
    Integer n_passangers;

    public Car(String licence, Account Driver){
        this.licence = licence;
        this.driver = driver;
    }

    void printDataCar(){
        System.Out.println('Licence '+ licence+ 'Name Driver '+driver.name);
    }
} 

File: Python/Car.py
from Account import Account

class Car:
    id = int
    license = str
    driver = Account('','','','')
    n_passangers = int 
    
    def __init__(self,license,driver,n_passangers):
        self.license = license
        self.driver = driver
        self.n_passangers = n_passangers